includes/polling/os/moxa-etherdevice.inc.php

```diff
@@ -10,12 +10,9 @@
  * the source code distribution for details.
  */
 
-
-// Moxa people enjoy creating MIBs for each model!
-if ($device['sysDescr'] == 'IKS-6726A-2GTXSFP-T') {
-    $mibmod = 'MOXA-IKS6726A-MIB';
-} elseif ($device['sysDescr'] == 'EDS-G508E-T') {
-    $mibmod = 'MOXA-EDSG508E-MIB';
-}
-$version = snmp_get($device, "firmwareVersion.0", "-OQvs", $mibmod);
-$hardware =  $device['sysDescr'];
+// switchModel:
+$hardware = snmp_get($device, $device['sysObjectID'].'.1.2.0', '-OQvs');
```

Comment:
Please convert these three snmp_get calls into snmp_get_multi_oid. See other files for examples.